Modules/_testcapimodule.c

```diff
@@ -3301,6 +3301,19 @@ getbuffer_with_null_view(PyObject* self, PyObject *obj)
     Py_RETURN_NONE;
 }
 
+/* test PyBuffer_SizeFromFormat() */
+static Py_ssize_t
+pybuffer_size_from_format(PyObject* self, PyObject *args)
+{
+    const char *format;
+
+    if (!PyArg_ParseTuple(args, "s:pybuffer_size_from_format",
+                          &format))
```

Comment:
nitpick: please add {...}, I know that old code doesn't respect PEP 7, but new code should respect it.